keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -2769,6 +2828,18 @@ Why should this KEP _not_ be implemented?
 
 ## Alternatives
 
+### Semantic Parameters instead of PodSchedulingContext
+
+When a DRA driver uses semantic parameters, there is no DRA driver controller
```

Comment:
I believe in the latest cut this has shifted a little? In the semantic parameter version, there is still a driver controller but its responsibility is to evaluate the claim parameters and produce the driver-neutral semantic request. Since we are skipping CEL for this (for now at least...). But the following text on "we might be able to remove PodSchedulingContext" looks right.

Reply:
I change the text into "there is no need for a DRA driver controller
which allocates the claim and no need for communication between scheduler and such a controller"

That leaves it open whether a controller is still needed for other purposes (will be defined in the "semantic parameters KEP" and may depend on whether users are allowed to create in-tree parameter objects directly) and just focuses on the aspect that is relevant here.